### PR TITLE
feat: 新增支持查询apikey使用情况

### PR DESCRIPTION
**Pull request by maq917**

### CODE REVIEW

File: config.template.json [hunk 1]
```diff
@@ -11,5 +11,7 @@
   "clear_current_session_command": "#clear session",
   "clear_all_sessions_command": "#clear all sessions",
   "chat_group_session_independent": false,
-  "single_chat_prefix": ["bot", "@bot"]
+  "single_chat_prefix": ["bot", "@bot"],
+  "query_key_command": "#query key",
```
Comment (iuiaoin) [inline config.template.json:15]: Thanks for the contribution🎉. Please also add relevant description in  in README [here](https://github.com/iuiaoin/wechat-gptbot#configuration) to help others understand your configuration～

**Review by iuiaoin — APPROVED** [review]

**Comment by iuiaoin** [comment]
And also don't forget to test your code and make sure it can work as you expect.

File: config.template.json (hunk 1 above)
Comment (maq917) [reply config.template.json:15]: README has been updated.

Comment (maq917) [reply config.template.json:15]: Yes, the code is able to work properly.

**Pull request merged**